login.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="ko">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="reset.css">
+</head>
+<body>
+    <div class="login-box">
+        <a href="/index.html">
+            <img class="login-pandalogo" src="/image/logo.png" alt="pandalogo">
+        </a>
+        <form action="/login.html" method="POST">
+            <div class="email">
+                <label for="email">이메일</label>
+                <input type="text" id="email" name="email" placeholder="email">
+            </div>
+            <div class="password">
+                <label for="password">비밀번호</label>
+                <input type="password" id="password" name="password" placeholder="password">
+            </div>
+            <button type="submit" name="login">로그인</button>
```

Comment:
❗️ 수정요청

버튼 태그의 name 속성은 태그의 value 속성과 같이 쓸때 필요한 요소입니다~
지금처럼 제출하는 submit 타입의 버튼의 경우 불필요합니다.
불필요한 속성은 지워주시는 것을 추천드려요!

[mdn](https://developer.mozilla.org/ko/docs/Web/HTML/Element/button#name)

```suggestion
            <button type="submit">로그인</button>
            // <button type='button' name='gender' value='female'>Female</button> 이런 경우 사용합니다.
```